.github/workflows/pr-checks.yaml

```diff
@@ -181,7 +181,12 @@ jobs:
 
           eval $(minikube docker-env)
           make release-manifests
+          make deploy-cert-manager
+          echo "Waiting 120s for cert-manager to start..."
+          sleep 120
```

Comment:
instead of this, could we wait on cert manager using something like

```
kubectl rollout status ....
```

Reply:
Done in 916e27d0e405455eaf20b87c8bbbf964591b0fc9.